blockchain/blockchain.go

```diff
@@ -1006,6 +1020,20 @@ func (bc *blockchain) commitBlock(blk *block.Block) error {
 	if err != nil {
 		return err
 	}
+	// write numActions into DB
+	numActionsTimer := bc.timerFactory.NewTimer("putNumActions")
```

Comment:
Can we encapsulate putNumActions and putTransferAmount into putBlock, and execute in one db batch